python/heterocl/devices.py

```diff
@@ -59,7 +59,7 @@ def __repr__(self):
 
 class Memory(object):
     """The base class for memory modules"""
-    def __init__(self, types, cap, channels):
+    def __init__(self, types, cap, channels=None, path=None):
```

Comment:
It seems like `path` is not used?

Reply:
It's used in SSD case, though SSD is not a memory in strict sense.

Reply:
It can have its own constructor, right? The base class is not using this variable.

Reply:
No. Sure, I can remove it.